cpp/include/cuspatial/utility/device_atomics.cuh

```diff
@@ -30,7 +32,7 @@ namespace detail {
  * @param val The value to compare
  * @return The old value stored in `addr`.
  */
-__device__ double atomicMin(double* addr, double val)
+__device__ inline double atomicMin(double* addr, double val)
```

Comment:
Shall we hold on to these changes until #561  is merged?

Reply:
I them needed to get this PR working.